### DISCUSSION TITLE
blog/id-spotify-now-playing

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#236] [General]
# blog/id-spotify-now-playing

API Spotify memberikan kita akses untuk mengetahui lagu yang sedang dimainkan di account kamu.

https://theodorusclarence.com/blog/id-spotify-now-playing

**Comment by apischiang** [comment]
thank you tutor nya ka!

**Comment by fiqryq** [comment]
mantap maass tengkyu

**Comment by nelsonlaidev** [comment]
Thank you! ❤️

**Comment by icarus** [comment]
Very insightful and helpful for a rookie like me hehe. I got stucked because I have an app directory instead of pages though, any workaround?

**Reply by TanasitISTG** [reply]
https://github.com/TanasitISTG/ttapp

I just did mine, try it!

**Comment by aryadzar** [comment]
Halo bang, terima kasih banyak tutornya ❤️❤️
